samples/bluetooth/hap_ha/prj.conf

```diff
@@ -0,0 +1,60 @@
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_PRIVACY=y
+
+CONFIG_BT_SMP=y
+CONFIG_BT_SMP_SC_PAIR_ONLY=y
+CONFIG_BT_SMP_MIN_ENC_KEY_SIZE=16
+
+CONFIG_BT_EXT_ADV=y
+CONFIG_BT_DEVICE_NAME="Hearing Aid sample"
+# Appearance: Generic Hearing aid (0x0A40)
+CONFIG_BT_DEVICE_APPEARANCE=2624
+
+CONFIG_BT_AUDIO=y
+CONFIG_BT_AUDIO_UNICAST_SERVER=y
+CONFIG_BT_ASCS_ASE_SNK_COUNT=1
+CONFIG_BT_ASCS_ASE_SRC_COUNT=1
+# Support an ISO channel per ASE
+CONFIG_BT_ISO_MAX_CHAN=2
+
+# Sink Contexts Supported: Unspecified, Conversational, Media, Live
+CONFIG_BT_PACS_SNK_CONTEXT=0x0047
+# Source Contexts Supported: Unspecified, Conversational
+CONFIG_BT_PACS_SRC_CONTEXT=0x0003
+
+CONFIG_BT_PAC_SNK_LOC=y
+CONFIG_BT_PAC_SRC_LOC=y
+
+CONFIG_BT_VCS=y
+CONFIG_BT_MICP_MIC_DEV=y
+CONFIG_BT_HAS=y
+CONFIG_BT_HAS_HEARING_AID_MONAURAL=y
+CONFIG_BT_HAS_PRESET_COUNT=4
+CONFIG_BT_HAS_PRESET_NAME_DYNAMIC=y
+CONFIG_BT_CAP_ACCEPTOR=y
+CONFIG_BT_CAP_ACCEPTOR_SET_MEMBER=n
+
+CONFIG_BT_AICS_MAX_INSTANCE_COUNT=2
+# The HA may expose an instance of MICS and an instance of AICS to control
+# the capture of ambient sound.
+CONFIG_BT_VCS_AICS_INSTANCE_COUNT=1
+# An HA may instantiate one or more instances of Audio Input Control Service
+# (AICS) to expose control of the gain of its inputs to a Volume Controller.
+CONFIG_BT_MICP_MIC_DEV_AICS_INSTANCE_COUNT=1
+
+# If the HA supports the Volume Balance feature (see Section 3.1) and the HA
+# is part of a Binaural Hearing Aid Set, the HA shall instantiate one instance
+# of Volume Offset Control Service (VOCS).
+CONFIG_BT_VOCS_MAX_INSTANCE_COUNT=1
+CONFIG_BT_VCS_VOCS_INSTANCE_COUNT=1
+
+CONFIG_BT_BAS=y
+CONFIG_BT_IAS=y
+CONFIG_BT_CSIS=y
```

Comment:
Isn't it only for the binaural hearing aid setup that we need to have CSIS? For mono and banded isn't that only a single device, and thus not required to be a set?

Reply:
Ah, that it's true. I did not catch that I have it in common .conf file still